src/java/org/apache/cassandra/config/Config.java

```diff
@@ -725,6 +723,25 @@ public static void setClientMode(boolean clientMode)
     public volatile SubnetGroups client_error_reporting_exclusions = new SubnetGroups();
     public volatile SubnetGroups internode_error_reporting_exclusions = new SubnetGroups();
 
+    public static final int DISABLED_GUARDRAIL = -1;
+    public volatile boolean guardrails_enabled = false;
+    public volatile int keyspaces_warn_threshold = DISABLED_GUARDRAIL;
+    public volatile int keyspaces_abort_threshold = DISABLED_GUARDRAIL;
+    public volatile int tables_warn_threshold = DISABLED_GUARDRAIL;
+    public volatile int tables_abort_threshold = DISABLED_GUARDRAIL;
```

Comment:
now that this is top level wondering if we should rename these 2 to be more clear; the rest I think are clear enough...

100% cool leaving out of this JIRA, just something to think of before may.

Reply:
The warning thresholds replace the previous properties `table_count_warn_threshold` and `keyspace_count_warn_threshold`, which were deprecated by CASSANDRA-17195. I think that the `_count_` part of the name isn't used in any other property on the file. I'd say that the new names seem more in line with other property names such as `tombstone_warn_threshold`, `cached_rows_warn_threshold`, `unlogged_batch_across_partitions_warn_threshold`, etc. 

In any case, I guess we'll revisit naming before the next release.